examples/django/django-project/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "watch": "concurrently \"npm run css:watch\"",
+    "css:watch": "sass -w app/assets/scss/main.scss app/static/css/main.css --load-path=node_modules/",
+    "css:build": "sass app/assets/scss/main.scss app/static/css/main.css --load-path=node_modules/"
+  },
+  "devDependencies": {
+    "concurrently": "^8.2.2",
+    "sass": "^1.75.0"
+  },
+  "dependencies": {
+    "@minvws/manon": "16.1.0",
+    "@minvws/manon-themes": "16.3.0-beta.2"
```

Comment:
As discussed before, this repo uses pnpm, but this example project doesn't. 

The reason we use an pnpm workspace is to easily use the local versions of `manon` and `manon-themes` in the docs and in example projects using the magic `"workspace:*"` version identifier. That won't work for **this** example project since it uses docker (even `npm link` might fail).

However:
- it might be useful to explicitly mention in the README that this example does **not** use pnpm and so won't include local changes to `manon`/`manon-themes`
- to avoid problems, we could consider replacing the `examples/*` glob in `pnpm-workspace.yml` with explicit entries (so we can omit this example)

Reply:
My main consideration of using npm was to minimize the layers of dependencies for the audience that might be unfamiliar with pnpm. In this first iteration I wanted to create the most minimal setup in order to setup a django project with manon. However, I do recognize the hypocrisy when this PR does introduce docker and concurrently into the mix, with the main reason of an easy setup and easy reproducibility. But that line of reasoning is the same for using pnpm, and will be further abstracted when it is handled by a docker container.

Do we recommend / is required to use manon together with pnpm? In that case it would be helpful if the example projects also resemble that. I do like the idea of having the local manon files be present in the example project ensuring you have the latest changes. Though when we start with a specific django package where manon is bundled it might not be necessary.

Thoughts?

Reply:
> I do recognize the hypocrisy when this PR does introduce docker and concurrently into the mix, with the main reason of an easy setup and easy reproducibility.

Providing a Dockerfile and docker-compose.yml increases reproducibility. They're optional, so if you're familiar with Django you can skip docker and run the app directly - no hypocrisy there as far as I'm concerned.

> Do we recommend / is required to use manon together with pnpm?

No, we use the pnpm workspace primarily to make it as easy as possible to work on `manon` and `manon-themes` while running the `docs` dev server. I.e. to avoid having to `npm link`.

So it's primarily meant for devs working on Manon or its docs.

I added the example apps to the pnpm workspace as a natural extension of that but perhaps I shouldn't have. It's useful for Manon devs but as you rightly point out it's probably confusing for Manon consumers.

Any library packages could potentially live in this repo too, and use the pnpm workspace to stay up date with Manon and get published alongside Manon updates.

Let's have an out-of-band chat about this soon. For now using npm is perfectly fine IMO.